src/components/Notification/NotificationItem/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { useNavigate } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+
+import Avatar from '@/components/Common/Avatar';
+import { MESSAGE } from '@/constants/notification';
+import { userAtom } from '@/recoil/user';
+import { PATH } from '@/routes/path';
+import { Notification } from '@/types/response';
+import { getElapsedTime } from '@/utils/getElapsedTime';
+
+import {
+  AuthorWrapper,
+  ContentWrapper,
+  MessageContainer,
+  MessageWrapper,
+  NotificationItemContainer,
+  TimeWrapper,
+  UserWrapper,
+} from './style';
+
+interface NotificationItemProps {
+  notification: Notification;
+}
+
+export default function NotificationItem({
+  notification,
+}: NotificationItemProps) {
+  const navigate = useNavigate();
+
+  const user = useRecoilValue(userAtom);
+  const { author, createdAt, post } = notification;
+  const notificationKeys = ['message', 'follow', 'like', 'comment'] as const;
+  const key = notificationKeys.find(
+    (notificationKey) => notificationKey in notification,
+  )!;
+
+  const handleClick = () => {
+    if (post) {
+      navigate(`${PATH.REVIEWDETAIL}/${post}`);
+
+      return;
+    }
```

Comment:
```suggestion
    if (post) {
      navigate(`${PATH.REVIEWDETAIL}/${post}`);
    } else {
      navigate(`${PATH.DIRECTMESSAGE}/${author._id}`);
    }
```

제가 방금 DM PR을 머지해서 pull dev 해주시고 path 설정해주시면
바로 DM 페이지로 넘어갈 거예요! 설정 부탁드리겠습니다 👍

Reply:
네 알겠습니다 이 부분도 추가할게요!!

Reply:
모두 수정했습니다 좋은 리뷰 감사합니다😁